### PR TITLE
`Effects` page and impure arguments

### LINKED ISSUE #62: Rewrite `Effects`'s paragraph with impure function argument
The page `Effects` has [this paragraph](https://github.com/flix/programming-flix/blob/6864d39b90922b78f8d3e5be7ae26b915e367166/src/content/Effects.js#L62) which explains "A higher-order function can also require that a function is impure.". But the syntax `~>` used in this example
```
def foreach(f: a ~> Unit, l: List[a]): Unit & Impure
```
is not allowed anymore. And the paragraph should be rewritten.

### PR DESCRIPTION
**Pull request by ninaandrup**
Closes #62

### CODE REVIEW

File: src/content/Effects.js [hunk 1]
```diff
@@ -226,7 +206,7 @@ Possible fixes:
                     <tr>
```
Comment (magnus-madsen) [inline src/content/Effects.js:206]: I would remove the entire line.

Comment (ninaandrup) [reply src/content/Effects.js:206]: I'm not sure which line you are refering to. Is it the in the table at the bottom of the page?

Comment (magnus-madsen) [reply src/content/Effects.js:206]: Yes, the row in the table.

Comment (ninaandrup) [reply src/content/Effects.js:206]: done

**Pull request merged**